app/src/main/res/layout/fragment_main.xml

```diff
@@ -21,33 +19,69 @@
 
     </androidx.coordinatorlayout.widget.CoordinatorLayout>
 
-    <FrameLayout
-        android:id="@+id/fragment_main_nav_tab_container"
+    <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="?attr/actionBarSize"
-        android:layout_gravity="bottom"
-        android:background="?attr/nav_tab_background_color"
-        android:elevation="6dp">
+        android:layout_height="wrap_content"
+        android:baselineAligned="false"
+        android:orientation="horizontal">
 
-        <org.wikipedia.navtab.NavTabLayout
-            android:id="@+id/fragment_main_nav_tab_layout"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
+        <FrameLayout
+            android:id="@+id/fragment_main_nav_tab_container"
+            android:layout_width="0dp"
+            android:layout_height="?attr/actionBarSize"
+            android:layout_gravity="bottom"
+            android:layout_weight="1"
             android:background="?attr/nav_tab_background_color"
```

Comment:
This background should now be applied to the parent view, and removed from here.